rusqlite_migration/src/errors.rs

```diff
@@ -70,6 +72,21 @@ impl From<rusqlite::Error> for Error {
     }
 }
 
+impl From<tokio_rusqlite::Error> for Error {
```

Comment:
I’m probably missing something here, but it’s unclear to me why CI passes with this. CI tests with the default features, where tokio-rusqlite is supposed to be inaccessible. And this file is not behind a feature flag so 🤷🏼

Reply:
True, that's surprising...

Reply:
I added a `#[cfg(feature = "async-tokio-rusqlite")]` for it anyway.